src/read.rs

```diff
@@ -0,0 +1,67 @@
+use std::io;
+use std::io::Write;
+use std::marker::PhantomData;
+use Event;
+
+pub struct ReadHtml<'a, I> where I: Iterator<Item=Event<'a>> + 'a {
```

Comment:
There's no logical relation between `ReadHtml` and `WriteHtml`, and therefore are confusing... We should either remove `WriteHtml` or rename `ReadHtml` to something else or do both!